docs/installation/server-config/security.rst

```diff
@@ -187,50 +196,51 @@ Standard ACLs
     - ``DESCRIBE`` permission on the ``__consumer_offsets`` topic.
 
 Input topics
-    The authenticated KSQL user requires ``DESCRIBE`` and ``READ`` permissions for each input topic. The topic should already exist
-    when KSQL is started.
+    An input topic is a topic that is imported into KSQL using a ``CREATE STREAM`` or ``CREATE TABLE``
+    statement. The topic should already exist when KSQL is started.
+
+    The authenticated KSQL user requires ``DESCRIBE`` and ``READ`` permissions for each input topic.
 
 Output topics
-    KSQL creates output topics when you run persistent CREATE TABLE AS STREAM and CREATE TABLE AS STREAM queries. The
-    authenticated KSQL user requires ``DESCRIBE`` and ``WRITE`` permissions on each output topic.
+    KSQL creates output topics when you run persistent ``CREATE TABLE AS SELECT`` or ``CREATE TABLE AS SELECT`` queries.
 
-    By default, KSQL will attempt to create any output topics that do not exist. To allow this the authenticated KSQL user requires
-    ``CREATE`` permissions on the ``CLUSTER`` resource type. Alternatively, topics can be created manual before running KSQL. To determine
-    the list of output topics and their required configuration, for example, partition count, replication factor,
-    retention policy, etc, you can run KSQL on an open cluster first.
+    The authenticated KSQL user requires ``DESCRIBE`` and ``WRITE`` permissions on each output topic.
 
-    If you want KSQL to create output topics, then the authenticated KSQL user should be granted ``CREATE`` permission on the ``CLUSTER`` resource type. KSQL
-    will create output topics whenever you are doing a persistent CTAS/CTAS query.
+    By default, KSQL will attempt to create any output topics that do not exist. To allow this, the authenticated KSQL user requires
+    ``CREATE`` permissions on the ``CLUSTER`` resource type. Alternatively, topics can be created manual before running KSQL. To determine
+    the list of output topics and their required configuration, (partition count, replication factor,
+    retention policy, etc), you can run KSQL on a Kafka cluster with no, or open, ACLs first.
```

Comment:
suggest: `...you can initially run KSQL on a Kafka cluster with none or open ACLs.`

Reply:
done